### PR TITLE
Issue 22790 create default variant

### LINKED ISSUE #22790: Create default variant
We need to create a `default variant`, so we need to:

**Create a new TaskUpgrade to:**

- Create a `default variant` in the variant table
- Add a new variant field to "contentlet_version_info", the default value should point to the `default variant`

### PR DESCRIPTION
**Pull request by freddyucv**
Creating a new TaskUpgrade to:

Create a default variant in the variant table
Add a new variant field to "contentlet_version_info", the default value should point to the default variant

### CODE REVIEW

File: dotCMS/src/main/resources/postgres.sql [hunk 1]
```diff
@@ -560,6 +560,8 @@ create table contentlet_version_info (
    locked_by varchar(100),
    locked_on timestamptz,
    version_ts timestamptz not null,
+   variant_id varchar(255),
```
Comment (jdotcms) [inline dotCMS/src/main/resources/postgres.sql:563]: it seems you have the same column twice, is that ok?

Comment (freddyucv) [reply dotCMS/src/main/resources/postgres.sql]: done https://github.com/dotCMS/core/pull/22868/commits/b8a61e219cbd169ac8faa56d1c828326886e2bdb

File: dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task220824CreateDefaultVariantTest.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            }
+        }
+
+        assertTrue( variantIdFound, "Should exists de variant field in contentlet_version_info");
+    }
+
+    /**
+     * Method to test: {@link Task220824CreateDefaultVariant#executeUpgrade()}
+     * when: the UT run
+     * Should: Create the default variant and add a new fiel in the contentlet_version_info
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task220824CreateDefaultVariantTest.java:73]: field*

**Review by jcastro-dotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/portlets/variant/business/VariantAPI.java [hunk 3]
```diff
@@ -5,6 +5,9 @@
 import java.util.Optional;
 
 public interface VariantAPI {
+
+    Variant DEFAULT_VARIANT = new Variant("1", "DEFAULT", false);
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotmarketing/portlets/variant/business/VariantAPI.java:9]: `new Variant("DEFAULT", "DEFAULT", false);`

File: dotCMS/src/main/resources/mssql.sql [hunk 4]
```diff
@@ -794,6 +794,7 @@ create table contentlet_version_info (
    locked_by NVARCHAR(100) null,
    locked_on datetimeoffset(3) null,
    version_ts datetimeoffset(3) not null,
+   variant_id NVARCHAR(255) default '1',
```
Comment (dsilvam) [inline dotCMS/src/main/resources/mssql.sql:797]: default 'DEFAULT'

File: dotCMS/src/main/resources/postgres.sql [hunk 5]
```diff
@@ -560,6 +560,7 @@ create table contentlet_version_info (
    locked_by varchar(100),
    locked_on timestamptz,
    version_ts timestamptz not null,
+   variant_id varchar(255) default '1',
```
Comment (dsilvam) [inline dotCMS/src/main/resources/postgres.sql:563]: default 'DEFAULT'

**Review by dsilvam — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/java/com/dotmarketing/portlets/variant/business/VariantAPI.java (hunk 3 above)
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotmarketing/portlets/variant/business/VariantAPI.java]: done https://github.com/dotCMS/core/pull/22868/commits/5218d2e674297b44e275be6486cb330985cf0506

File: dotCMS/src/main/resources/mssql.sql (hunk 4 above)
Comment (freddyucv) [reply dotCMS/src/main/resources/mssql.sql]: done https://github.com/dotCMS/core/pull/22868/commits/5218d2e674297b44e275be6486cb330985cf0506

File: dotCMS/src/main/resources/postgres.sql (hunk 5 above)
Comment (freddyucv) [reply dotCMS/src/main/resources/postgres.sql]: done https://github.com/dotCMS/core/pull/22868/commits/5218d2e674297b44e275be6486cb330985cf0506

**Review by dsilvam — APPROVED** [review]

**Pull request merged**